config/sources/families/include/rockchip64_common.inc

```diff
@@ -38,7 +38,7 @@ elif [[ $BOARD == rock64 ]]; then
 	BOOT_USE_MAINLINE_ATF=yes
 	BOOT_SOC=rk3328
 
-elif [[ $BOOTCONFIG == *3328* ]]; then
+elif [[ $BOOTCONFIG == *3328* || $BOARD == firefly-m1 ]]; then
```

Comment:
Is there any particular reason you want to use u-boot v2017.09 for this board?
I would like to remove this branch/scenario as soon as possible. At this point only Z28 Pro is still using it.

Reply:
> Is there any particular reason you want to use u-boot v2017.09 for this board?
> I would like to remove this branch/scenario as soon as possible. At this point only Z28 Pro is still using it.

At the moment, only this option works fully. When using u-boot-2020.07, USB does not work on the running system. I need time to find the cause and fix it.

Reply:
Does it work with mainline u-boot in your own images?
If so, maybe we need to adjust something in our u-boot build process/configuration?

Reply:
> Does it work with mainline u-boot in your own images?

In my builds with u-boot-2020.10, USB does not work either (I have a guess of the reason, but there is not enough time to implement the fix and create patches yet).